pkg/kubelet/status/status_manager_test.go

```diff
@@ -49,6 +49,12 @@ import (
 	"k8s.io/kubernetes/pkg/kubelet/util"
 )
 
+type mutablePodManager interface {
+	AddPod(*v1.Pod)
+	UpdatePod(*v1.Pod)
+	DeletePod(*v1.Pod)
```

Comment:
Other areas of the Kubelet use `RemovePod` terminology... Probably makes sense here.

Reply:
I see 7 mentions of RemovePod, and slightly more for pod manager's DeletePod.  Kubelet config loop uses `DeletePods` to mean a "pod has been requested for deletion", and "RemovePods" to mean "pod is gone".

Agree Remove should be used, I'll add a new commit.